### DISCUSSION TITLE
One API rule for CRUD operations?

### DISCUSSION THREAD

**Discussion by peterhijma** [pocketbase/pocketbase#3286] [Q&A]
I see myself using the same rule for view, list, create, update and delete all the time. Would it be an idea to also be able to set one rule for all instead of (error prone) copy pasting?

**Comment by ganigeorgiev** [comment]
I don't think it will be a good idea as it may complicate and clutter the UI.

A lot of different scenarios could arise (eg. someone may want only list, view and delete to have the same rule) and it would just create one more way to do the same thing and it feels unnecessary in this case.

**Comment by peterhijma** [comment]
Alright, thank you for the answer! 👍
